RijsMuseum/app/src/main/java/com/example/rijsmuseum/fragments/DetailsFragment.kt

```diff
@@ -32,12 +31,17 @@ class DetailsFragment : Fragment(R.layout.fragment_collection_details) {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         titleText = view.findViewById(R.id.fragmentTitle)
```

Comment:
You don't use titleText outside of the scope of this method, then it is not needed as a property of the class.